test/basic-querying.test.js

```diff
@@ -275,7 +289,7 @@ describe('basic-querying', function() {
       });
     });
 
-    it('should support "or" that is satisfied', function(done) {
+    bdd.itIf(!isCassandraConnector, 'should support "or" that is satisfied', function(done) {
```

Comment:
Cassandra doesn't support `OR` operator. It makes sense to skip. Does the Cassandra connector raise an error is an `OR` operator is used?